suite/tests/client-interface/drx-scattergather.c

```diff
@@ -152,13 +152,28 @@ test_avx2_vgatherqpd(uint32_t *ref_sparse_test_buf, uint32_t *test_idx32_vec,
 #    ifdef __AVX512F__
 /* See comment above. */
 void
-test_avx512_restore_mask_fault(uint32_t *ref_sparse_test_buf, uint32_t *test_idx32_vec);
+test_avx512_restore_gather_mask_fault(uint32_t *ref_sparse_test_buf,
```

Comment:
These should all be static, right?

Reply:
Done.

Reply:
These are the asm functions and not in the same module.

Reply:
OK, I see.